flight-icons/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hashicorp/flight-icons",
-  "version": "0.0.6-beta",
+  "version": "0.0.8-beta",
```

Comment:
cc @randallmorey there's no change to the svg's in this version bump, but it moves the icons into a folder `@hashicorp/flight-icons/icons/`. i made that change for now, because we may want the `export-go/` to dump into that folder (this ticket: https://github.com/hashicorp/flight/issues/84), while keeping other files such as README/CONTRIBUTING in root `flight-icons/`

just FYI, there's no reason to bump your package rn, but the namespacing could change in the future. i know you mentioned that before in https://github.com/hashicorp/flight/issues/90

it could also be called `dist/` instead of `icons/`, is that generally more common? just curious

Reply:
What if it were named `flight-icons/` for simpler namespacing?  I know it's redundant.  It just makes it that much easier to import.  With a folder named just `icons/`, we'd have to use either `flight-icons/icons/*` or `icons/*`, but the latter is too generic to be useful.

Reply:
thanks - sounds good, that makes sense. or, i can explore using `files` again? i'll extract this issue to https://github.com/hashicorp/flight/issues/142 and test separately